### PR TITLE
[Profiler] Use tabular design for HTTP request/headers

### PR DESCRIPTION
**Pull request by ostrolucky**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | yes
| Related tickets | partially #245
| Documentation   |
| License         | MIT

#### What's in this PR and why?

Improved profiler design. Check [Before](https://user-images.githubusercontent.com/496233/69001989-3605b500-08e8-11ea-90b2-9a1c7b6d0b3f.png) and [After](https://user-images.githubusercontent.com/496233/69001956-dc9d8600-08e7-11ea-91a0-f44dad85a106.png). Also check linked issue.

#### Checklist

- [x] Updated CHANGELOG.md to describe BC breaks / deprecations | new feature | bugfix

#### Out of scope
- `<t(d|h)>` text trimming. I just want to make this simple first time. This can be added in future, at this time horizontal scrolling works perfect here.
- Syntax highlighting for body. Should be done in separate PR, I want to minimze scope here which will help review and merge this faster.

### CODE REVIEW

**Comment by dbu** [comment]
awesome, thanks! and completely agree to keep syntax highlighting or more complicated layouting tweak to separate pull requests to make it easier to review.

can you please rebase? there is a conflict (i guess because #354 adding a link to the debug token)

**Comment by ostrolucky** [comment]
I've slightly changed the design of #354, because its label style didn't fit in my feature, hopefully that's fine. 

Screenshot 2019-11-18 at 10 32 54

**Comment by dbu** [comment]
i like it! @dbrumann as you contributed the profile link, i ask you if you are happy with this too or want a change?

**Comment by dbrumann** [comment]
These changes look great. :+1: I like it. I will have a look at the code changes later today, but consider the end result approved

**Comment by ostrolucky** [comment]
It just accured to me that maybe we could just highlight any link in header table? Because if you notice, there is a x-debug-token-link header there already. That would be more universal

**Comment by dbu** [comment]
as this is a very specific thing to symfony, i like the link at the top.

additionally detecting urls in headervalues and link them sounds like a good idea (for a separate pull request)

**Comment by ostrolucky** [comment]
Allright. We can also reduce complexity further by getting token link from header to display in twig, then Profiler no longer needs to store it separately.

**Comment by dbu** [comment]
indeed. i see you removed the token from the display and from the profiler data (i think thats good, the token code does not help and is annoying to copy-paste if its part of a link anyways, better copy-paste from the headers). want to do that in this PR or separately?

**Comment by ostrolucky** [comment]
I can do it here

**Comment by ostrolucky** [comment]
done

File: src/Resources/views/http_message.html.twig [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{% set hasReachedBody = false %}
+{% set content = '' %}
+{% set data = data|split("\n")|slice(1) %}
+{% set xdebugTokenLink = data|filter(v => 'x-debug-token-link:' in v|lower)|first|split(':')|slice(1)|join(':')|trim %}
```
Comment (dbu) [inline src/Resources/views/http_message.html.twig]: could you instead split with a limit of 1, to not break up the url and then have to re-join it? would be more elegant imho. `data|filter(v => 'x-debug-token-link:' in v|lower)|first|split(':', 2)|slice(1)|trim`

Comment (ostrolucky) [reply src/Resources/views/http_message.html.twig]: I came up with even better simplification. WDYT?

**Comment by dbu** [comment]
nice!

**Pull request merged**